diff --git a/lib/util/dialog.js b/lib/util/dialog.js
index 380b804..604900b 100644
--- a/lib/util/dialog.js
+++ b/lib/util/dialog.js
@@ -33,14 +33,14 @@
         }
       });
       inp.focus();
-      CodeMirror.on(inp, "blur", close);
-    } else if (button = dialog.getElementsByTagName("button")[0]) {
+      //CodeMirror.on(inp, "blur", close);
+    } else if (button == dialog.getElementsByTagName("button")[0]) {
       CodeMirror.on(button, "click", function() {
-        close();
-        me.focus();
+        //close();
+        //me.focus();
       });
       button.focus();
-      CodeMirror.on(button, "blur", close);
+      //CodeMirror.on(button, "blur", close);
     }
     return close;
   });
